README.md

```diff
@@ -12,6 +12,11 @@ version](https://img.shields.io/github/r-package/v/epiforecasts/scoringutils)
 downloads](http://cranlogs.r-pkg.org/badges/grand-total/scoringutils)](https://cran.r-project.org/package=scoringutils)
 <!-- badges: end -->
 
+**Note**: This documentation refers to the development version of
```

Comment:
For this to be safe the README needs to be triggered to render when the description is updated.